src/spark_rapids_ml/knn.py

```diff
@@ -166,21 +190,14 @@ class NearestNeighbors(
     >>> topk = 2
     >>> gpu_knn = NearestNeighbors().setInputCol("features").setIdCol("id").setK(topk)
     >>> gpu_model = gpu_knn.fit(data_df)
-    >>> (query_df, data_df, knn_df) = gpu_model.kneighbors(query_df)
+    >>> (data_df, query_df, knn_df) = gpu_model.kneighbors(query_df)
```

Comment:
Why reverse?

Reply:
Observed pyspark LSH API implicitly puts item vectors before queries in the [example code](https://spark.apache.org/docs/2.2.3/ml-features.html#approximate-nearest-neighbor-search): 

(1) model.approxNearestNeighbors(dfA, key, 2).show()
(2) dfA is defined before dfB. 

The (1) implies the dfA corresponds to the item vectors, and the (2) implies item vectors were defined before query vectors. Looking for a better way to keep consistent with the example code.